posit_workbench_best_practice_with_r.md

```diff
@@ -0,0 +1,485 @@
+# Best Practice with R in Posit Workbench
+
+Original author(s): Andrew Patterson (Jumping Rivers), Terry McLaughlin
+
+Last updated: January 2023
+
+## Background
+
+[Posit Team](https://posit.co/products/enterprise/team/) enterprise applications have been deployed for [Public Health Scotland (PHS)](https://publichealthscotland.scot/) on the [Microsoft Azure](https://azure.microsoft.com/en-gb/) cloud computing platform. The platform has been designed and implemented as a high-performance, high-availability analytical environment to support the work of Public Health Scotland. However, despite the computing power available, we need to be mindful at all times that this is a shared resource with finite capacity, and as such, we each have individual responsibility to ensure our code is as optimised and efficient as possible, and that we use the Posit Team applications correctly and appropriately, to ensure what we all benefit from this resource equally. Using inefficient code can cause your analysis to take longer or even cause the session to become unresponsive.
+
+## Purpose
+
+This document aims to offer guidance to users on good practices for efficient and effective use of R and Posit Applications (in particular Posit Workbench) on Microsoft Azure.
+
```

Comment:
Shameless promotion for the Knowledge Base. Ensuring the link between different resources is brought together (especially if that will become a repository for more in the future). 
```suggestion
This document aims to offer guidance to users on good practices for efficient and effective use of R and Posit Applications (in particular Posit Workbench) on Microsoft Azure. This guidance provides a background and specific advice, for more guidance and training, please visit the [PHS Data Science Knowledge Base](https://public-health-scotland.github.io/knowledge-base/).
```